libs/sdk-core/src/breez_services.rs

```diff
@@ -425,8 +426,25 @@ impl BreezServices {
     }
 
     pub async fn reverse_swap_info(&self) -> Result<ReverseSwapInfo> {
-        let rsi = crate::boltzswap::reverse_swap_info().await?;
-        return Ok(rsi);
+        crate::boltzswap::reverse_swap_info().await
```

Comment:
Shouldn't we add the `reverse_swap_info` to the `ReverseSwapperAPI` trait implemented by BoltzServer?
It is good to have traits for APIs not only because we will support different swap implementations but also for the code to be testable.

Reply:
Done.